src/LegacyCallHandler.tsx

```diff
@@ -531,7 +533,7 @@ export default class LegacyCallHandler extends EventEmitter {
         };
         if (audio) {
             if (this.audioPromises.has(audioId)) {
-                this.audioPromises.set(audioId, this.audioPromises.get(audioId).then(pauseAudio));
+                this.audioPromises.set(audioId, this.audioPromises.get(audioId)!.then(pauseAudio));
```

Comment:
Only my opinion: Instead of using `!` here (and in other places in this PR) it could be avoided by storing the promise into a variable and test if the variable is set. In this way the code can be more robust if someone changes the condition for instance.

Reply:
This is really a workaround for a Typescript bug, this if statement clause is a `Map::has` within which `Map::get` would not be nullable for the same key. Hopefully fixed soon, this is the approach we use elsewhere.